File: CarSpecAngular/carspec/src/app/core/models/interfaces/DataProcessingDtos/apiResponseDtos/brochure-import-review-dto.ts
import { AuditChangeReviewDto } from "./audit-change-review-dto";

export interface BrochureImportReviewDto {
  importBatchId: number;
  status: string;
  model1: any;
  auditChanges: AuditChangeReviewDto[];
  warnings: string[];
}

File: CarSpecAPI/Data/Models/DataExtractionModel/BrochureImportReviewDto.cs
namespace CarSpecAPI.Data.Models.DataExtractionModel
{
    public class BrochureImportReviewDto
    {
        public int ImportBatchId { get; set; }

        public string Status { get; set; } = string.Empty;

        public BrochureExtractionDto Model1 { get; set; } = new();

        public List<AuditChangeReviewDto> AuditChanges { get; set; } = new();

        public List<string> Warnings { get; set; } = new();
    }
}
